test/Libraries/Microsoft.Extensions.ObjectPool.DependencyInjection.Tests/Microsoft.Extensions.ObjectPool.DependencyInjection.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Extensions.ObjectPool.DependencyInjection.Tests</AssemblyName>
+    <RootNamespace>Microsoft.Extensions.ObjectPool.Test</RootNamespace>
+    <Description>Unit tests for Microsoft.Extensions.ObjectPool.DependencyInjection.</Description>
```

Comment:
You could do this:
```suggestion
    <Description>Tests for $(MSBuildProjectName.Replace('.Tests', ''))</Description>
```

Reply:
I'd rather keep it human-readable, like we have in the other project files in this repo.